Update SSH configuration docs (#3013)
codenvy/che-docs: modules/administration-guide/pages/configuring-a-user-namespace.adoc

----
che.eclipse.org/sync-retain-on-delete: "true"
----

> For example, to mount a default SSH Config into every workspace, you need to create the ConfigMap:

Reviewer: ```suggestion
For example, to mount a default SSH configuration into every workspace, you must create a ConfigMap:
```

+
====
[source,yaml,subs="+attributes,+quotes"]
----
kind: ConfigMap
apiVersion: v1
metadata:
  name: ssh-config-configmap
  namespace: eclipse-che
  labels:
    app.kubernetes.io/component: workspaces-config
    app.kubernetes.io/part-of: che.eclipse.org
  annotations:
    controller.devfile.io/mount-as: subpath
    controller.devfile.io/mount-path: /etc/ssh/ssh_config.d/
data:
  ssh.conf: <ssh config content>
----
====
+

> This ConfigMap will propagate the SSH Config as an extension to the existing default SSH Config by the `Include /etc/ssh/ssh_config.d/*.conf` argument, see: link:https://man.openbsd.org/ssh_config#Include[].

Reviewer: ```suggestion
This **ConfigMap** propagates the SSH configuration as an extension to the existing default SSH configuration by using the `Include /etc/ssh/ssh_config.d/*.conf` argument. For more information, review the link:https://man.openbsd.org/ssh_config#Include[Include] definition.
```

+

> See the link:https://github.com/devfile/devworkspace-operator/blob/main/docs/additional-configuration.adoc#automatically-mounting-volumes-configmaps-and-secrets[mounting volumes, configmaps, and secrets]

Reviewer: ```suggestion
For other labels and annotations, see link:https://github.com/devfile/devworkspace-operator/blob/main/docs/additional-configuration.adoc#automatically-mounting-volumes-configmaps-and-secrets[Mounting volumes, configmaps, and secrets].
```

> for other possible labels and annotations.

Reviewer: ```suggestion
```
